### DISCUSSION TITLE
Don't have access to slides.

### DISCUSSION THREAD

**Discussion by shreyansh-dash** [mrdbourke/pytorch-deep-learning#217] [General]
Have been trying to access the course slides in the repo, but shows invalid pdf, can't render rich display
![Screenshot 2022-12-16 200822](https://user-images.githubusercontent.com/79135001/208122015-89afea51-9ffb-4079-ae8e-065579479df2.png)

**Comment by mrdbourke** [comment]
Hi @shreyansh-dash,

Sometimes GitHub lags out and can't offer the slides to download.

You can find all of the slides here on Dropbox: https://www.dropbox.com/sh/crcw2s5vsca7h7f/AACiI9HQ1Epu-Dq1bOGqVxzAa?dl=0

Let me know if it works :)
